kernel/src/main/scala/cats/kernel/instances/StaticMethods.scala

```diff
@@ -5,15 +5,6 @@ import scala.collection.mutable
 
 object StaticMethods {
 
-  def initMutableMap[K, V](m: Map[K, V]): mutable.Map[K, V] = {
-    val result = mutable.Map.empty[K, V]
-    m.foreach { case (k, v) => result(k) = v }
-    result
-  }
-
-  def wrapMutableMap[K, V](m: mutable.Map[K, V]): Map[K, V] =
```

Comment:
if you remove this, it becomes unusable from the outside. That was the reason for it, I think, to allow using it from say `algebra`.